write.go

```diff
@@ -0,0 +1,233 @@
+package rct
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math"
+)
+
+/*
+ * See https://github.com/do-gooder/rctpower_writesupport?tab=readme-ov-file#usage
+ */
+
+// SetSocStrategy sets the SOC strategy (power_mng.soc_strategy) with the given ENUM value
+func (c *Connection) SetSocStrategy(strategy uint8) error {
+	if strategy > SOCTargetSchedule {
+		return fmt.Errorf("invalid SOC strategy value: %d", strategy)
+	}
+
+	builder := NewDatagramBuilder()
+	builder.Build(&Datagram{
+		Cmd:  Write,
+		Id:   PowerMngSocStrategy,
+		Data: []byte{strategy},
+	})
+
+	_, err := c.Send(builder)
+	if err != nil {
```

Comment:
```suggestion
	if _, err := c.Send(builder); err != nil {
```

Ich finde generell dieses Error Wrapping in Libraries nicht sinnvoll. Als Entwickler weiss ich, welche Funktion ich aufgerufen habe! In Applikationen sieht das anders aus, da sollte dann aber die Applikation diese Entscheidung treffen.

Reply:
generell: braucht es diese ganzen Funktionen oder nicht einfach nur `Write` und ggf. noch Schreibbeispiele in /example?

Reply:
Ich überlasse die Entscheidung dir: Soll ich die `write.go` entfernen?

Reply:
Das kann ich nicht entscheiden. Für evcc ist es einfach, die `Write` Funktion direkt zu nutzen und auch die Daten vorher zu validieren.

Reply:
Dann würde ich es lassen.
Wer sich etwas unsicher ist, nutzt die Methoden.
Wer das nicht möchte, kann die Werte direkt schreiben.